.eslintrc

```diff
@@ -0,0 +1,253 @@
+ecmaFeatures: {}
+rules:
+  no-alert: 0
+  no-array-constructor: 0
+  no-bitwise: 0
+  no-caller: 0
+  no-catch-shadow: 0
+  no-class-assign: 0
+  no-cond-assign: 2
+  no-console: 2
+  no-const-assign: 0
+  no-constant-condition: 2
+  no-continue: 0
+  no-control-regex: 2
+  no-debugger: 2
+  no-delete-var: 2
+  no-div-regex: 0
+  no-dupe-keys: 2
+  no-dupe-args: 2
+  no-duplicate-case: 2
+  no-else-return: 0
+  no-empty: 2
+  no-empty-character-class: 2
+  no-empty-label: 0
+  no-eq-null: 0
+  no-eval: 0
+  no-ex-assign: 2
+  no-extend-native: 0
+  no-extra-bind: 0
+  no-extra-boolean-cast: 2
+  no-extra-parens: 0
+  no-extra-semi: 2
+  no-fallthrough: 2
+  no-floating-decimal: 0
+  no-func-assign: 2
+  no-implicit-coercion: 0
+  no-implied-eval: 0
+  no-inline-comments: 0
+  no-inner-declarations:
+  - 2
+  - functions
+  no-invalid-regexp: 2
+  no-invalid-this: 0
+  no-irregular-whitespace: 2
+  no-iterator: 0
+  no-label-var: 0
+  no-labels: 0
+  no-lone-blocks: 0
+  no-lonely-if: 0
+  no-loop-func: 0
+  no-mixed-requires:
+  - 0
+  - false
+  no-mixed-spaces-and-tabs:
+  - 2
```

Comment:
This will probably trigger pointless errors since I do, in fact, mix tabs and spaces on purpose. I use tabs for indentation and spaces for alignment, a style that surprisingly few tools have caught up to, despite the number of people using it.
So, if that's the case, we should change that to a 0...